initrd/bin/seal-hotpkey

```diff
@@ -66,6 +69,7 @@ counter_value=1
 enable_usb
 # While making sure the key is inserted, capture the status so we can check how
 # many PIN attempts remain
+DEBUG "Calling hotp_verification info..."
 if ! hotp_token_info="$(hotp_verification info)" ; then
```

Comment:
If the DO_WITH_DEBUG spam is reverted:

`if ! hotp_token_info="$(DO_WITH_DEBUG hotp_verification info)"; then`